homeassistant/components/smarty/fan.py

```diff
@@ -0,0 +1,133 @@
+"""
+Platform to control a Salda Smarty XP/XV ventilation unit.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/fan.smarty/
```

Comment:
We don't include the url in the docstring anymore. Just keep the very first line.